src/submission/models.py

```diff
@@ -364,6 +370,85 @@ def validate(self, value, model_instance):
                     raise
 
 
+class ArticleSearchManager(BaseSearchManagerMixin):
+
+    def search(self, *args, **kwargs):
+        queryset = super().search(*args, **kwargs)
+        return queryset.filter(
+            date_published__lte=timezone.now()
```

Comment:
Elsewhere we also check that the article is in `STAGE_PUBLISHED`, could also check here?